src/PrestaShopBundle/Resources/views/Admin/Layout/core_javascript.html.twig

```diff
@@ -21,5 +21,17 @@
  * @author    PrestaShop SA and Contributors <contact@prestashop.com>
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
- *#}
-{{ component('EmployeeDropdown') }}
+ * #}
+{% block core_javascripts %}
```

Comment:
I just wonder is it possible to override/extend a block that is defined in an include? Can you test this by forcing an extension in a page just to be sure?

If not possible you just need to put the block in the layout and include the file:
```twig
{% block core_javascripts %}
  {{ include('core_javascripts.html.twig') }}
{% endblock %}
```